SPECS/cairo/cairo.spec

```diff
@@ -85,31 +89,22 @@ This package contains tools for working with the cairo graphics library.
 %autosetup -p1
 
 %build
-%configure \
-        --disable-gl \
-        --disable-gtk-doc \
-        --disable-static \
-        --disable-symbol-lookup \
-        --enable-ft \
-        --enable-gobject \
-        --enable-pdf \
-        --enable-ps \
-        --enable-svg \
-        --enable-tee \
-        --enable-win32=no \
-        --enable-xlib
-
-sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
-sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
-%make_build
 
+%meson \
+  -Dfreetype=enabled \
+  -Dfontconfig=enabled \
+  -Dglib=enabled \
+  -Dgtk_doc=true \
+  -Dspectre=disabled \
+  -Dsymbol-lookup=disabled \
+  -Dtee=enabled \
+  -Dtests=disabled \
+  -Dxcb=enabled \
+  -Dxlib=enabled \
+  %{nil}
+%meson_build
```

Comment:
This looks like a big configuration change. Are we not removing features we already provided with the earlier config or introducing new ones we don't need? For instance what happened to `--enable-svg` or `--enable-ps`? I see we also switched `gtk_doc` to be enabled now (I think).

Reply:
I have taken another pass at the spec file and summarized the changes in the changelog. Let me know if I missed anything. Thanks for bringing this to my attention!